src/common/remote/mod.rs

```diff
@@ -1,10 +1,14 @@
 pub mod grpc;
 
 use std::sync::atomic::{AtomicI64, Ordering};
+use crate::api::error::Error::WrongGrpcAddress;
+
+use crate::api::error::Result;
 
 // odd by client request id.
 const SEQUENCE_INITIAL_VALUE: i64 = 1;
 const SEQUENCE_DELTA: i64 = 2;
+const IPV4: &str = "ipv4";
```

Comment:
Move the code here to the top? The definition of continuous variables that do not affect `SEQUENCE`

Reply:
@CherishCai I just put constant together with another ones. I've moved it before these "sequence" ones.